afew/filters/DKIMValidityFilter.py

```diff
@@ -0,0 +1,57 @@
+# coding=utf-8
+from __future__ import print_function, absolute_import, unicode_literals
+
+#
+# Copyright (c) Amadeusz Zolnowski <aidecoe@aidecoe.name>
+#
+# Permission to use, copy, modify, and distribute this software for any
+# purpose with or without fee is hereby granted, provided that the above
+# copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+# WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+# ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+# ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+# OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
```

Comment:
Can you use [SPDX](https://wiki.spdx.org/view/Technical_Team/SPDX_Meta_Tags), and simply add a single
```
# SPDX-License-Identifier: ISC
```
instead of the license text?
I still have a PR lying around fixing this in other places as well :-)

Reply:
@aidecoe see https://github.com/afewmail/afew/pull/171 for the PR.